config/database.yml

```diff
@@ -1,27 +1,32 @@
 default: &default
   adapter: postgresql
+  # url format: postgres://username:password@localhost:5432/database_name
+  url: <%= ENV["DATABASE_URL"] %>
+  # It's best to set DATABASE_URL to configure database details. That overrides the following settings
+  username: <%= ENV["HOSTEDGPT_DB_USERNAME"] || "hostedgpt_user" %>
+  password: <%= ENV["HOSTEDGPT_DB_PASSWORD"] %>
+  host: <%= ENV["HOSTEDGPT_DB_HOST"] %>
+  port: <%= ENV["HOSTEDGPT_DB_PORT"] %>
+  database: <%= ENV["HOSTEDGPT_DB_NAME"] || "hostedgpt" %>
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  port: <%= ENV['HOSTEDGPT_DATABASE_PORT'] || 5432 %>
```

Comment:
changing the names of a lot of these env vars for consistency. This would be a breaking change for anyone with an existing setup. I can have it look at the old env vars too so that it wont break anyone's existing setup or we can make people update and provide instructions for how to do so.